master/internal/api_task.go

```diff
@@ -30,3 +31,25 @@ func (a *apiServer) GetTask(
 			errors.Wrapf(err, "error fetching task %s from database", req.TaskId)
 	}
 }
+
+func (a *apiServer) GetTaskConfig(
+	ctx context.Context, req *apiv1.GetTaskConfigRequest,
+) (resp *apiv1.GetTaskConfigResponse, err error) {
+	if err := a.canDoActionsOnTask(ctx, model.TaskID(req.TaskId),
+		expauth.AuthZProvider.Get().CanGetExperimentArtifacts); err != nil {
+		return nil, err
+	}
+
+	t := &taskv1.Task{}
+	switch err := a.m.db.QueryProto("get_task", t, req.TaskId); {
+	case errors.Is(err, db.ErrNotFound):
+		return nil, api.NotFoundErrs("task", req.TaskId, true)
+	default:
+		config, err := json.Marshal(t.Config)
```

Comment:
can we just select it as a string? save ourselves a lot of unmarshaling? this gets really costly in the get experiments endpoint